main.py

```diff
@@ -1,19 +1,27 @@
 # !/usr/bin/env python3
 
-from flask import render_template
+from flask import render_template, request
 
 from __init__ import app
 
 
 @app.route("/", methods=["GET"])
 def index():
     app.logger.info("index(): GET /")
-    template_filename = "index.html"
+    return render_template("index.html")
+
+
+@app.route("/search", methods=["GET"])
+def search():
+    app.logger.info("search(): GET /search")
```

Comment:
```suggestion
    app.logger.info(f"search(): GET {request.full_path}")
```
こうすると、queryも込みでlogger出力してくれます

Reply:
ありがとうございます。